tasks/postcss.ts

```diff
@@ -9,28 +9,31 @@ export = function(grunt: IGrunt) {
 	grunt.loadNpmTasks('grunt-postcss');
 
 	const distDirectory = grunt.config.get<string>('distDirectory');
+	const devDirectory = grunt.config.get<string>('devDirectory');
 
-	const moduleProcessors: any = [
-		postCssImport,
-		postCssNext({
-			features: {
-				autoprefixer: {
-					browsers: [
-						'last 2 versions',
-						'ie >= 10'
-					]
+	function moduleProcessors(dest: string, cwd = '') {
+		return [
+			postCssImport,
+			postCssNext({
+				features: {
+					autoprefixer: {
+						browsers: [
+							'last 2 versions',
+							'ie >= 10'
+						]
+					}
 				}
-			}
-		}),
-		postCssModules({
-			generateScopedName: '[hash:base64:8]',
-			getJSON: function(cssFileName: string, json: JSON) {
-				const outputPath = path.resolve(distDirectory, path.relative('src', cssFileName));
-				const newFilePath = outputPath.replace(/.css$/, '.js');
-				fs.writeFileSync(newFilePath, umdWrapper(JSON.stringify(json)));
-			}
-		})
-	];
+			}),
+			postCssModules({
+				generateScopedName: '[hash:base64:8]',
```

Comment:
For `dev` you probably want to change the scope name back the the BEM style too?